src/api/requestsApi.tsx

```diff
@@ -56,7 +56,7 @@ export interface RPARequest {
   fullPartTime?: "Full" | "Part";
   salaryLow?: number;
   salaryHigh?: number;
-  telework?: "Yes" | "No";
+  telework?: "None" | "Regular-recurring" | "Situational";
```

Comment:
Because this is now moved to the main section, it is no longer dependent on the type of request, so validateRequest function needs updated to move it from each of the existing sections to the new location